### PR TITLE
feat: Add `CreateCandidate` use case with provider `breezy-hr`

### PR DESCRIPTION
**Pull request by martinalbert**
## Description
Implements map for provider `breezy-hr`.

### blockers:
Unable to upload CV.

## Motivation and Context
ATS integrations

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by janhalama** [comment]
@martinalbert I fixed the conflict when creating candidate with email which already applied to the job and removed CV upload which is not supported at the moment by OneSDK. I also restructured the code into operations.

**Comment by martinalbert** [comment]
Looks great, thanks!

File: grid/recruitment/candidates/maps/breezy-hr.suma [hunk 1]
```diff
@@ -160,3 +108,136 @@ operation MapBreezyHRError {
...
+
+  links = {}
+
+  candidate = args.candidate
+
+  tmp = (candidate.links || []).forEach(link => {
+    if (supportedSocialProfiles.includes(link.name)) {
+      links[link.name] = link.url
+    } else {
+      links['globe'] = link.url
```
Comment (janhalama) [inline grid/recruitment/candidates/maps/breezy-hr.suma:126]: issue: you are rewriting globe with the all the social link urls that are not in the supported list. I propose to add not supported links to `custom_attributes`.

File: grid/recruitment/candidates/maps/breezy-hr.suma [hunk 2]
```diff
@@ -160,3 +108,136 @@ operation MapBreezyHRError {
...
+        title = 'Conflict'
+        detail = 'Candidate with email ' + args.candidate.email + ' already applied for the job.' 
+        code = 'Conflict'
+        rateLimit = call MapRateLimit(headers = headers)
+      }
+
+      return {}
+    }
+
+    response 400 {
```
Comment (janhalama) [inline grid/recruitment/candidates/maps/breezy-hr.suma:221]: issue: this needs to be handled as error

**Review by janhalama — COMMENTED** [review]
I have one issue with my code and one in the link mapping.

**Review by janhalama — APPROVED** [review]
LGTM 👍

**Pull request merged**